gcs/README.md

```diff
@@ -0,0 +1,246 @@
+    This workspace should be populated from a Secret in your TaskRuns and PipelineRuns.
+* **output**: A workspace for this Task to copy the files from GCS in to.
+
+### Parameters
+
+* **name**: The name of the file or directory that will be written to the output workspace. (_required_)
+* **location**: The address, including "gs://", of the bucket you'd like to copy from. (_required_)
+* **typeDir**: Set this to "true" if the object you are copying is a directory. (_default_: "false")
+* **serviceAccountPath**: The path to the service account credential file in your credentials workspace. (_default_: "service\_account.json")
+
+## `gcs-upload`
+
+A `Task` that uploads files or directories from a Workspace to a GCS bucket.
+
+### Workspaces
+
+* **credentials**: A workspace that contains a service account key as a JSON file.
+    This workspace should be populated from a Secret in your TaskRuns and PipelineRuns.
+* **source**: A workspace where files will be uploaded from.
+
+### Parameters
+
+* **name**: The path to files or directories relative to the source workspace that you'd like to upload. (_required_)
+* **location**: The address (including "gs://") where you'd like to upload files to. (_required_)
+* **serviceAccountPath**: The path to the service account credential file in your credentials workspace. (_default_: "service\_account.json")
+
+## `gcs-create-bucket`
+
+A `Task` that creates a new GCS bucket.
+
+### Workspaces
+
+* **credentials**: A workspace that contains a service account key as a JSON file.
+    This workspace should be populated from a Secret in your TaskRuns and PipelineRuns.
+
+### Parameters
+
+* **bucketName**: The name of the bucket (including "gs://") to create. (_required_)
+* **project**: The project with which your bucket will be associated. (_required_)
+* **storageClass**: The storage class for the new bucket. STANDARD, NEARLINE, COLDLINE, or ARCHIVE. (_default_: "STANDARD")
+* **region**: The region, dual-region, or multi-region for the new bucket. (_default_: "")
+* **uniformAccess**: Set this to "true" if the bucket should be created with bucket-level permissions instead of Access Control Lists. (_default_: "false")
+* **serviceAccountPath**: The path to the service account credential file in your credentials workspace. (_default_: "service\_account.json")
+
+## `gcs-delete-bucket`
+
+A `Task` that deletes a GCS bucket.
+
+### Workspaces
+
+* **credentials**: A workspace that contains a service account key as a JSON file.
+    This workspace should be populated from a Secret in your TaskRuns and PipelineRuns.
+
+### Parameters
+
+* **bucketName**: The name of the bucket (including "gs://") to create. (_required_)
+* **serviceAccountPath**: The path to the service account credential file in your credentials workspace. (_default_: "service\_account.json")
+
+## Usage
+
+### `gcs-download`
+
+This pipeline uses the gcs-download Task to fetch a directory.
```

Comment:
maybe we want to have a "samples" dir that contains these so over time we could at the very least make sure the syntax is correct + continues to match the Tasks